content/en/docs/contributing/release-process.md

```diff
@@ -261,7 +266,8 @@ page if a step is missing or if it is outdated.
         ```sh
         # Must be run from the cert-manger folder.
         export GITHUB_TOKEN=*your-token*
-        go install k8s.io/release/cmd/release-notes@v0.7.0
```

Comment:
I removed this line because we already install the tool in the first 3 steps of the release process.